test-utils/src/main/java/org/javaee7/ServerOperations.java

```diff
@@ -98,7 +98,7 @@ public static void addCertificateToContainerTrustStore(Certificate clientCertifi
                 logger.info("Using domain \"" + domain + "\" obtained from server. If this is not correct use -Dpayara_domain to override.");
             }
 
-            Path cacertsPath = gfHomePath.resolve("glassfish/domains/" + domain + "/config/cacerts.jks");
+            Path cacertsPath = gfHomePath.resolve("glassfish/domains/" + domain + "/config/cacerts.p12");
```

Comment:
You also need to change the key store type on line 113, as it's still trying to open it as JKS.
`keyStore = KeyStore.getInstance("JKS");`